cluster-autoscaler/cloudprovider/aws/aws_wrapper.go

```diff
@@ -261,3 +149,58 @@ func (m *autoScalingWrapper) getAutoscalingGroupNamesByTags(kvs map[string]strin
 
 	return asgNames, nil
 }
+
+func (m *awsWrapper) getInstanceTypeByLT(launchTemplate *launchTemplate) (string, error) {
```

Comment:
i think it would be nice to just name this `getInstanceTypeByLaunchTemplate`, just to make it clear

Reply:
++, also renamed `getInstaceTypeByLCNames`